src/application/renderers/isf.js

```diff
@@ -87,6 +85,10 @@ async function setupModule(module) {
   const renderer = new ISFRenderer(isfContext);
   renderer.loadSource(fragmentShader, vertexShader);
 
+  if (!renderer.valid) {
+    throw renderer.errorWithCorrectedLines;
```

Comment:
Is `errorWithCorrectedLines` new? I've not seen this before.

Reply:
Yes, I added this to ISFRenderer as the original was just giving back the errorLine of the first error and the error-message still contains the errorLines from the parsed shader, which is not helpful at all. 

See https://github.com/vcync/interactive-shader-format-js/blob/fixedIsfParserAndImageBitmap/src/ISFRenderer.js#L55